tests/QueryTest.php

```diff
@@ -158,7 +158,7 @@ public function testSetRawQuery(): void
     {
         $query = new Query();
 
-        $params = ['query' => 'test'];
+        $params = ['query' => ['term' => ['title' => 'test']]];
```

Comment:
Nope, the thing here is that in this test we are checking the `Query::toArray()` method, but I guess it wouldn't work in a functional test since it's not specifying which query type to use.